### PR TITLE
Basic sync operation backfill

### PR DESCRIPTION
**Pull request by Brendonovich**
Adds a system for generating sync operations from an existing library's data. It's not resumable or anything and isn't actually integrated into the app nicely but it's a start.
This is 1/3 of ENG-1613.

### CODE REVIEW

**Comment by Brendonovich** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #2101** <a href="https://app.graphite.dev/github/pr/spacedriveapp/spacedrive/2101?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/spacedriveapp/spacedrive/2101?utm_source=stack-comment).

File: core/prisma/schema.prisma [hunk 1]
```diff
@@ -168,7 +168,7 @@ model FilePath {
   object_id Int?
   object    Object? @relation(fields: [object_id], references: [id], onDelete: SetNull)
 
-  key_id Int? // replacement for encryption
+  // key_id Int? // replacement for encryption
```
Comment (utkubakir) [inline core/prisma/schema.prisma:171]: do we need migration for this or nah?

Comment (Brendonovich) [reply core/prisma/schema.prisma:171]: Ah yep, will make a migration - we're just not using this field rn so I'm disabling it

Comment (Brendonovich) [reply core/prisma/schema.prisma]: nvm just re-enabled it since it's not a big deal

**Review by oscartbeaumont — APPROVED** [review]

**Pull request merged**